include/onnxruntime/core/common/status.h

```diff
@@ -86,14 +86,14 @@ class Status {
   Status(StatusCategory category, int code);
 
   Status(const Status& other)
-      : state_((other.state_ == nullptr) ? nullptr : std::make_unique<State>(*other.state_)) {}
+      : state_((other.state_ == nullptr) ? nullptr : new State(*other.state_)) {}
```

Comment:
The last time I did this,  Pranav rejected my change.

Reply:
I wonder why?

Reply:
https://github.com/microsoft/onnxruntime/pull/1730

Reply:
@snnn The context was different. IIRC in your change, it was a factory function that was returning a raw ptr.

Reply:
Basically, if you are using operator 'new', you're outdated.